src/backend/executor/execMain.c

```diff
@@ -1809,91 +1809,6 @@ InitPlan(QueryDesc *queryDesc, int eflags)
 
```

Comment:
I guess commit b9d3ce9a9f0c5bcf209d3eb44d017e155306b58d is related to GPDB commit 744dffaf1119fbc55ed5bc7d067365b4ee877e52(https://github.com/greenplum-db/gpdb-archive/commit/744dffaf1119fbc55ed5bc7d067365b4ee877e52). I am also not sure which case fixed this related these commits.

But since CBDB does not cherry-pick commit 744dffaf1119fbc55ed5bc7d067365b4ee877e52. I think it is better to directly remove commit b9d3ce9a9f0c5bcf209d3eb44d017e155306b58d (instead of adding a revert commit).

This is because there are non-consecutive cherry-pick GPDB commits in CBDB. We will use the TITLE to confirm whether the current commit has been cherry-picked. If there is revert commit with a non-cherry-picked commit, it may make the current developer unsure whether the current GPDB commit is needed.

So how about direct delete the commit `fix incorrect RTE's rellockmode in the process of translating...` in current PR?

Reply:
Thanks, I agree. It looks very strange to cherry pick and then delete a commit.